src/components/comment/Comment.tsx

```diff
@@ -14,10 +16,10 @@ const Comment = ({
     LikesCount,
     likeComment,
 }: IComment) => (
-    <li>
-        <div>
+    <li className={classes.comment}>
```

Comment:
name the top level class for a component `root`
this is a convention we use in wcs

Reply:
for all components